### PR TITLE
`property-case` rule

### PR DESCRIPTION
**Pull request by evilebottnawi**

### CODE REVIEW

File: src/rules/property-case/README.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+```css
+a {
+  border-Radius: 5px;
+}
+```
+
+```css
+a { 
+  -WEBKIT-animation-duration: 3s;
```
Comment (jeddy3) [inline src/rules/property-case/README.md]: Nice to see vendor prefixes considered!

File: src/rules/property-case/index.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        "lower",
+        "upper",
+      ],
+    })
+    if (!validOptions) { return }
+
+    root.walkDecls(decl => {
+      const prop = decl.prop
+      // Ignore sass interpolation and css variables
+      if (/#{.+}/.test(decl.toString()) || cssWordIsVariable(prop)) { return }
```
Comment (jeddy3) [inline src/rules/property-case/index.js]: Perhaps a good time to move the sass interpolation into a utility as we're using it in quite a few places. Perhaps `containsInterpolation(string)`?

Comment (evilebottnawi) [inline src/rules/property-case/index.js]: @jeddy3 many rules check on interpolation, i think we should do this in other PR.

Comment (jeddy3) [inline src/rules/property-case/index.js]: Agreed.

**Comment by davidtheclark** [comment]
Lgtm. Just needs a rebase.

**Comment by evilebottnawi** [comment]
@davidtheclark done

**Comment by jeddy3** [comment]
@evilebottnawi Flawless PR :) Thanks!

**Pull request merged**